chinese/freq.py

```diff
@@ -0,0 +1,54 @@
+from aqt.utils import showInfo
+from os.path import dirname, join, realpath
+import codecs
+import re
+import os
+
+def lookup_frequency(hanzi):
+
+    # Complicated math by ernop
+    levels={200:'very basic',
+        100:'basic',
+        50:'very common',
+        25:'common',
+        13:'uncommon',
+        7:'rare',
+        2:'very rare',
+        0:'obscure'}
+
+    slevels=sorted(levels.items(),key=lambda x:x[0]*-1)
+    words=sorted([w.upper() for w in levels.values()],key=lambda x:-1*len(x))
+
+    # Declare path to the corpus file
```

Comment:
I don't really believe in comments, which I realise is heresy. If the code is clear enough, don't comment. If it's not clear, rewrite it. Here, it's clear enough.